ui/src/chat/ChatChannel.tsx

```diff
@@ -26,6 +26,7 @@ import ChatScrollerPlaceholder from '@/chat/ChatScoller/ChatScrollerPlaceholder'
 
 function ChatChannel({ title }: ViewProps) {
   const navigate = useNavigate();
+  const joinRef = new URLSearchParams(window.location.search).get('joinref');
```

Comment:
can this be accomplished with `useSearchParams`?

https://reactrouter.com/zh/main/hooks/use-search-params

Reply:
What will this get us in this case?

Reply:
Consider this a nit, it feels more idiomatic but is certainly not a merge-blocker. If it doesn't get us anything then skip it